projects/Mallard/android/app/src/main/java/com/guardian/editions/ophan/InstallationIdHelper.java

```diff
@@ -37,7 +39,6 @@ private synchronized String readId(@NonNull Context context) {
         return sID;
     }
 
-    @NotNull
```

Comment:
Removed it because it does require to import another library

Reply:
Fantastic, thanks for your help.